Functional.Maybe/MaybeAsync.cs

```diff
@@ -30,6 +31,15 @@ public static async Task<T> OrElse<T>(this Task<Maybe<T>> @this, Func<T> orElse)
 		{
 			var res = await @this;
 			return res.HasValue ? res.Value : orElse();
+		} 
+		public static async Task<Maybe<T>> ToMaybeAsync<T>(this Task<T> task)
+		{
+			return (await task).ToMaybe();
```

Comment:
Why not use an arrow function here?

Reply:
No specific reason. I will convert it to arrows - this one and the one below. I will follow the formatting of `ToNullable()` that is already in the codebase.

Reply:
Please check the new formatting - it follows the one from `ToNullable()` function